test/e2e/healthcheck_run_test.go

```diff
@@ -203,4 +203,20 @@ var _ = Describe("Podman healthcheck run", func() {
 		inspect = podmanTest.InspectContainer("hc")
 		Expect(inspect[0].State.Healthcheck.Status).To(Equal("healthy"))
 	})
+
+	It("stopping and then starting a container with healthcheck cmd", func() {
+		session := podmanTest.Podman([]string{"run", "-dt", "--name", "hc", "--health-cmd", "[\"ls\", \"/foo\"]", ALPINE, "top"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
```

Comment:
Exit() is not defined in this file, you need to import `. "github.com/onsi/gomega/gexec"` so the test still compiles. That is why the linter fails
(Note sure if anyone actually runs these tests on the old branches)